bin/polyply

```diff
@@ -129,6 +128,12 @@ def main():
     topology_group.add_argument('-ign', dest='ignore', type=str,
                                 help='molecules to ignore when building structure',
                                 default=[], nargs='+')
+    topology_group.add_argument('-cycles', dest='cycles', type=str,
+                                help='molecule names of cyclic molecules',
+                                default=[], nargs='+')
```

Comment:
You can, but at the moment the algorithm only allows for linear cycles and at most one per molecule. So I thought it would be a nicer choice to set this actively.